oauth2-authorization-server/src/test/java/org/springframework/security/oauth2/server/authorization/web/OAuth2TokenEndpointFilterTests.java

```diff
@@ -247,6 +247,57 @@ public void doFilterWhenAuthorizationCodeTokenRequestValidThenAccessTokenRespons
 		assertThat(accessTokenResult.getScopes()).isEqualTo(accessToken.getScopes());
 	}
 
+	@Test
+	public void doFilterWhenAuthorizationCodeAndClientIdTokenRequestValidThenAccessTokenResponse() throws Exception {
```

Comment:
This test is the same as [doFilterWhenAuthorizationCodeTokenRequestValidThenAccessTokenResponse()](https://github.com/spring-projects-experimental/spring-authorization-server/blob/master/oauth2-authorization-server/src/test/java/org/springframework/security/oauth2/server/authorization/web/OAuth2TokenEndpointFilterTests.java#L200)